docs/automated-tests/troubleshooting/use-partial-run.md

```diff
@@ -1,47 +1,47 @@
 ---
 title: Use Partial Test Run
-page_title: How to Use Partial Test Run to Troubleshoot?
-description: "Use the Test Studio feature to execute only part of a test. Partial test execution allows you to run only certain steps from a test and continue recording steps from that point." 
+page_title: How to Use Partial Test Run to Troubleshoot
+description: "Use the Partial Test Run feature in Test Studio to execute only a test segment. Partial test execution allows you to run only certain steps from a test and continue recording steps from that point." 
 position: 4
 ---
 # Use Partial Test Runs to Troubleshoot a Failure
 
-In many cases it is quite useful to execute only part of a test - when you need to insert few new steps in the middle of a test, or when you need to debug a failing step. Test Studio provides solution for these cases with its powerful ability to combine the test execution of few steps and leave the page in that state in recording session.
+Sometimes you need to execute only part of a test - when you need to insert few new steps in the middle of a test, or when you need to debug a failing step. The solution is the partial test run feature.
 
-Let’s take a look at this useful feature with the following example and practical demonstration.
+This article is a practical demonstration of the partial test run.
 
 ## Record the Scenario
 
-For this tutorial we use a sample application. The task is to load the sample page and to verify a _button_ is enabled.
+In this tutorial, we use a sample application. The task is to load the sample page and verify that a _button_ is `enabled`.
 
 ![Sample web app](/img/automated-tests/troubleshooting/use-partial-run/1Scenario.png)
 
-We’ve already launched <a href="/automated-tests/recording/overview#start-a-recording-session" target="_blank">Test Studio’s recorder and recorded</a> the following test steps:
+We’ve already launched <a href="/automated-tests/recording/overview#start-a-recording-session" target="_blank">Test Studio’s recorder</a> and recorded the following test steps:
 
 ![Steps in test](/img/automated-tests/troubleshooting/use-partial-run/2RecordedTest.png)
 
 ## Execute the Test and Explore the Results
 
-Upon <a href="/automated-tests/test-execution/quick-execution" target="_blank">execution of this sample test</a>, we expect it to pass successfully. However, the __verification in step 2. fails__ and the failure message clearly states that the ___button_ state is not enabled__. The _Images_ section shows that the correct page is loaded, which brings no hint for identifying the cause of the failure.
```

Comment:
- In "execution of this sample test" the anchor should be "execution" because the linked article doesn't provide any details about "this sample test".
- "enabled" is a value and must be rendered in 'monochrome'